app/components/shared/progress_card_component.html.erb

```diff
@@ -4,17 +4,37 @@
-      <li><%= render progress_card_step_component(step: SubmissionPresenter::SUPPLEMENTAL_FILES_STEP, label: 'Supplemental files uploaded') %></li>
-      <li><%= render progress_card_step_component(step: SubmissionPresenter::PERMISSION_FILES_STEP, label: 'Permission files uploaded') %></li>
-      <li><%= render progress_card_step_component(step: SubmissionPresenter::RIGHTS_STEP, label: 'License terms applied') %></li>
-      <li><%= render progress_card_step_component(step: SubmissionPresenter::SUBMITTED_STEP, label: 'Submitted to Registrar', step_at: submitted_at) %></li>
-    </ul>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::CITATION_STEP, label: 'Citation details verified') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::CITATION_STEP, label: 'Citation details verified') %>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::ABSTRACT_STEP, label: 'Abstract provided') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::ABSTRACT_STEP, label: 'Abstract provided') %></li>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::FORMAT_STEP, label: 'Format reviewed') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::FORMAT_STEP, label: 'Format reviewed') %></li>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::DISSERTATION_STEP, label: 'Dissertation uploaded') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::DISSERTATION_STEP, label: 'Dissertation uploaded') %></li>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::SUPPLEMENTAL_FILES_STEP, label: 'Supplemental files uploaded') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::SUPPLEMENTAL_FILES_STEP, label: 'Supplemental files uploaded') %></li>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::PERMISSION_FILES_STEP, label: 'Permission files uploaded') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::PERMISSION_FILES_STEP, label: 'Permission files uploaded') %></li>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::RIGHTS_STEP, label: 'License terms applied') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::RIGHTS_STEP, label: 'License terms applied') %></li>
+      <% end %>
+      <%= tag.li 'aria-label': aria_label_for_step(step: SubmissionPresenter::SUBMITTED_STEP, label: 'Submitted to Registrar') do %>
+        <%= render progress_card_step_component(step: SubmissionPresenter::SUBMITTED_STEP, label: 'Submitted to Registrar', step_at: submitted_at) %></li>
+      <% end %>
```

Comment:
For follow-up work, I wonder if the progress card step component could have a new flag to invoke the "surrounded by list-item element" behavior?

Reply:
That's sort of what I'm thinking. I started down that path and realized it might require a bit more reach than expected and didn't want to hold this up.